LICENSE.save

```diff
@@ -0,0 +1,14 @@
+Copyright 2019-2021 by Sven Kreiss and contributors. All rights reserved.
```

Comment:
accidental commit?

Reply:
Thanks for pointing that out! I accidentally committed those files, that came from the merge tool.
I deleted the files.